components/atoms/address/address.js

```diff
@@ -5,25 +5,33 @@
 // React
 import React from 'react'
 import { string } from 'prop-types'
-
+import { Text, SIZE } from 'components'
 // Style
 import styled from 'styled-components'
 
+const AddressText = props => {
```

Comment:
@mhsnprvr3 @Ali-sharafi-git I dont see the change in this file that I pointed out in the last comment.....